e2e/portalicious/components/TableComponent.ts

```diff
@@ -205,9 +210,11 @@ class TableComponent {
   async filterColumnByDate({
     columnName,
     day,
+    range = 'Date',
   }: {
     columnName: string;
     day: number;
+    range?: string;
```

Comment:
I think:
- range can be renamed to something like `filterMode`. What do you think?
- Als I think it can a required argument without the default value